src/Raven.Server/Documents/Indexes/IndexDefinitionBase.cs

```diff
@@ -195,11 +195,17 @@ private static unsafe void EncryptStream(StorageEnvironmentOptions options, Memo
             var nonce = Sodium.GenerateRandomBuffer(Sodium.crypto_aead_xchacha20poly1305_ietf_npubbytes()); // 192-bit
             var encryptedData = new byte[data.Length + Sodium.crypto_aead_xchacha20poly1305_ietf_abytes()]; // data length + 128-bit mac 
 
+            fixed (byte* ctx = Sodium.Context)
```

Comment:
That is the same context as everything else, let's use a different (named) context in this way